database/migrations/2021_11_17_074221_add_owner_id_to_multiplayer_playlist_items.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddOwnerIdToMultiplayerPlaylistItems extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('multiplayer_playlist_items', function (Blueprint $table) {
+            $table->unsignedInteger('owner_id')->after('room_id')->default(0);
```

Comment:
a nullable would make more sense than a magic default

Reply:
How do I make it not nullable with no default? I was getting test errors: https://github.com/ppy/osu-web/runs/4236637200?check_suite_focus=true

Reply:
add a user in the factory (`database/factories/Multiplayer/PlaylistItemFactory.php`)

Reply:
Not sure how I missed this one 💦